heartbeat/monitors/active/http/simple_transp.go

```diff
@@ -147,13 +146,30 @@ func (t *SimpleTransport) writeRequest(conn net.Conn, req *http.Request) error {
 	return err
 }
 
+// comboConnReadCloser wraps a ReadCloser that is backed by
```

Comment:
Will this need a rebase? I remember almost same code went into an other PR.

Reply:
Yeah, I'll rebase it soon.